tests/Options/MessagesOptionsTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DotTest\Rbac\Guard\Options;
+
+use Dot\Rbac\Guard\Options\MessagesOptions;
+use PHPUnit\Framework\TestCase;
+
+class MessagesOptionsTest extends TestCase
+{
+    protected MessagesOptions $subject;
+    protected array $messageArray = ['testMessage'];
+    protected int $messageKey     = 0;
+
+    public function setUp(): void
+    {
+        $this->subject = new MessagesOptions();
+    }
+
+    public function testGetMessages()
+    {
+        $this->subject->setMessages($this->messageArray);
+        $result = $this->subject->getMessages();
+
+        $this->assertIsArray($result);
+    }
+
+    public function testGetMessage()
```

Comment:
1. Please add missing return type declaration.
2. Please rewrite this test as following:
```
$this->assertEmpty($this->subject->getMessage(1000));
$this->assertIsString($this->subject->getMessage(MessagesOptions::UNAUTHORIZED));
```
(This way, you could also remove the `messageArray` and `messageKey` properties)